india_banking_connector/host_to_host/doctype/base_host.py

```
@@ -0,0 +1,303 @@
+					)
+					status = client.upload(
+						str(payment_file_path), "/" + destination_path
+					)
+					if status:
+						uploaded_count += 1
+						field_uploaded = f"uploaded_{mot}"
+						frappe.db.set_value(
+							"H2H Payment Log", log_id, field_uploaded, 1
+						)
+						frappe.db.commit()
+				except Exception:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)